### PR TITLE
SEARCH-1351 Add support for filter queries in SQL requests.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

File: src/main/java/org/alfresco/rest/api/search/SearchSQLApiWebscript.java [hunk 1]
```diff
@@ -142,6 +142,7 @@ public SearchParameters buildSearchParameters(SearchSQLQuery searchQuery)
             Locale locale = new Locale(action);
             sparams.addLocale(locale);
         });
+        searchQuery.getFilterQueries().forEach(sparams::addFilterQuery);
```
Comment (michaelsuzukisagi) [inline src/main/java/org/alfresco/rest/api/search/SearchSQLApiWebscript.java]: What if searchQuery.getFilterQueries is null

**Comment by michaelsuzukisagi** [comment]
Do we have a test that covers the filter query is working?

File: src/main/java/org/alfresco/rest/api/search/SearchSQLApiWebscript.java (hunk 1 above)
Comment (tpage-alfresco) [reply src/main/java/org/alfresco/rest/api/search/SearchSQLApiWebscript.java]: The pattern in the constructor prevents this: https://github.com/Alfresco/alfresco-remote-api/blob/68c9a6602f11d17989522d43c1477f270bc8d693/src/main/java/org/alfresco/rest/api/search/model/SearchSQLQuery.java

**Comment by tpage-alfresco** [comment]
The tests for this are here: https://git.alfresco.com/tas/alfresco-tas-restapi-test/merge_requests/100

**Pull request merged**